examples/produce_assembly_urdf/README.md

```
@@ -0,0 +1,57 @@
+its meshes and switches the declaration over.
+
+```shell
+pc convert assembly -t assy robot
+```
+
+
+## Assemblies
+
+### robot
+<table><tr>
+<td valign=top><a href="robot.urdf"><img src="././robot.svg" style="width: auto; height: auto; max-width: 200px; max-height: 200px;"></a></td>
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)